<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoadCoins</class>
 <widget class="QWidget" name="LoadCoins">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>359</width>
    <height>389</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>10</y>
     <width>371</width>
     <height>381</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(25, 90, 35);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSub">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/* Default QPushButton style */
QPushButton {
    color: rgb(255, 255, 255);
    font: 700 15pt &quot;Segoe UI&quot;;
    background-color: rgb(20, 57, 102);
    border: 2px solid white;
    border-radius: 5px;
}

/* Style for QPushButton when hovered */
QPushButton:hover {
    background-color: rgb(35, 82, 127); /* Slightly lighter color on hover */
}

/* Style for QPushButton when pressed */
QPushButton:pressed {
    background-color: rgb(10, 32, 57); /* Darker color when pressed */
}
</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="currentCoinsText">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 700 15pt &quot;Segoe UI&quot;;
background-color: rgb(30, 90, 90);</string>
   </property>
   <property name="text">
    <string>Current Total ($):</string>
   </property>
  </widget>
  <widget class="QLabel" name="currentCoins">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>30</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 700 15pt &quot;Segoe UI&quot;;
background-color: rgb(30, 90, 90);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="AddingCoins">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>160</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 700 15pt &quot;Segoe UI&quot;;
background-color: rgb(30, 90, 90);</string>
   </property>
   <property name="text">
    <string>100</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonLoad">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>250</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/* Default QPushButton style */
QPushButton {
    color: rgb(255, 255, 255);
    font: 700 15pt &quot;Segoe UI&quot;;
    background-color: rgb(20, 57, 102);
    border: 2px solid white;
    border-radius: 5px;
}

/* Style for QPushButton when hovered */
QPushButton:hover {
    background-color: rgb(35, 82, 127); /* Slightly lighter color on hover */
}

/* Style for QPushButton when pressed */
QPushButton:pressed {
    background-color: rgb(10, 32, 57); /* Darker color when pressed */
}
</string>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>310</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/* Default QPushButton style */
QPushButton {
    color: rgb(255, 255, 255);
    font: 700 20pt &quot;Segoe UI&quot;;
    background-color: rgb(134, 47, 32);
    border: 2px solid white;
    border-radius: 5px;
}

/* Style for QPushButton when hovered */
QPushButton:hover {
    background-color: rgb(160, 57, 40); /* Slightly lighter color on hover */
}

/* Style for QPushButton when pressed */
QPushButton:pressed {
    background-color: rgb(100, 35, 25); /* Darker color when pressed */
}
</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonAdd">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>160</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/* Default QPushButton style */
QPushButton {
    color: rgb(255, 255, 255);
    font: 700 15pt &quot;Segoe UI&quot;;
    background-color: rgb(20, 57, 102);
    border: 2px solid white;
    border-radius: 5px;
}

/* Style for QPushButton when hovered */
QPushButton:hover {
    background-color: rgb(35, 82, 127); /* Slightly lighter color on hover */
}

/* Style for QPushButton when pressed */
QPushButton:pressed {
    background-color: rgb(10, 32, 57); /* Darker color when pressed */
}
color: rgb(255, 255, 255);
font: 700 15pt &quot;Segoe UI&quot;;
background-color: rgb(177, 0, 0);
border: 2px solid white;
 border-radius: 5px;</string>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
